test/EasyPost/CarrierAccountTest.php

```diff
@@ -38,6 +42,42 @@ public function testCreate()
         $carrierAccount->delete(); // Delete the carrier account once it's done being tested.
     }
 
+    /**
+     * Test creating a carrier account with custom workflow.
+     */
+    public function testCreateWithCustomWorkflow()
+    {
+        VCR::insertCassette('carrier_accounts/create_with_custom_workflow.yml');
```

Comment:
Please use the new `TestUtil::setupCassette()` function instead:

```suggestion
        TestUtil::setupCassette('carrier_accounts/createWithCustomWorkflow.yml');
```

The filename also needs to follow convention